src/components/AddResourceModal/AddFood/AddFood.jsx

```diff
@@ -79,38 +78,27 @@ const AddFood = ({
         justifyContent: 'center'
       }}
     >
-      {isMobile ? (
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: isMobile ? 'flex-end' : 'center',
+          justifyContent: isMobile ? null : 'center',
+          padding: isMobile ? '0px 20px 10px' : '20px 0',
+          height: isMobile ? '88px': '64px',
+          backgroundColor: '#FF9A55',
+        }}>
         <Typography
-          display="flex"
-          flexDirection="row"
-          alignItems="flex-end"
-          padding="0px 20px 10px"
-          height="88px"
-          backgroundColor="#FF9A55"
-          color="common.white"
-        >
+          sx={{
+            color: 'common.white',
+            textAlign: isMobile ? null : 'center',
```

Comment:
nit: if all properties are null on mobile, can we put the check on a higher level?
```js
<Typography sx={
  isMobile ? ({}) : ({ color: 'common.white', ....})
}> 
```

Reply:
Unfortunately the exception in the two highlighted scenarios is the color, which is set for both desktop/mobile. An option to address the majority of properties being exclusive could be to use a spread operator to group the check output into a single check operation. For example:
```javascript
<Typography
  sx={{
    color: 'common.white',
    ...(isMobile ? {} : {
      textAlign: 'center',
      fontFamily: 'Inter',
      fontWeight: 600,
      fontSize: 20.16
    })
}}>
  Add a Food Resource
</Typography>
```

I tested it and it works, what do you think?

I'll get this merged ahead of the response, but want to make sure I don't leave this `nit` thread hanging.